web/modules/custom/nfafmis/nfafmis.views.inc

```diff
@@ -37,4 +37,68 @@ function nfafmis_views_query_alter(ViewExecutable $view, QueryPluginBase $query)
       }
     }
   }
+  elseif ($view->storage->get('id') === 'cfr_lists') {
+    // Get the current user's assigned management unit.
+    $nfafmis_service_utility = \Drupal::service('nfafmis_service.utility');
+    $range = $nfafmis_service_utility->getUserManagementUnit();
+
+    if ($range) {
+      // Add a join to the management unit data table.
+      $definition = [
+        'table' => 'taxonomy_term__management_unit',
+        'type' => 'INNER',
+        'field' => 'entity_id',
+        'left_table' => 'taxonomy_term_field_data',
+        'left_field' => 'tid',
+      ];
+
+      $join = Drupal::service('plugin.manager.views.join')->createInstance('standard', $definition);
+      $query->addRelationship('taxonomy_term__management_unit', $join, 'person');
+
+      // Limit CFRs to those that are in the user's management unit.
+      $query->addWhere('conditions', 'taxonomy_term__management_unit.management_unit_target_id ', $range->id(), '=');
+    }
+  }
+}
+
+/**
+ * Implements hook_views_pre_render().
+ *
+ * @param \Drupal\views\ViewExecutable $view
+ */
+function nfafmis_views_pre_render(ViewExecutable $view) {
+  $view->element['#attached']['library'][] = 'nfafmis/nfafmis';
```

Comment:
This would add this library to all views, wouldn't it?

Reply:
Yes it would. That's a legacy hook that I just moved from one fie to another. I didn't spot the issue. Will fix it

Reply:
Changed this so it's included only in the farmer_main_tab view.